### ISSUE TITLE
App project

### ISSUE THREAD

**Issue by mitchiexx4** [mitchiexx4/girls-group-for-IT#1]
Hiii guys

**Comment by wendylamptey** [comment]
Heyy

**Comment by SL12Golden** [comment]
Hii

**Comment by wendylamptey** [comment]
Now I don't think the network is a problem so that's be active here

**Comment by wendylamptey** [comment]
We all heard what sir said today so let's back to them and do a better job.

**Comment by SL12Golden** [comment]
https://form.jotform.com/240784484466569 
This is the Form that will be used for the Cafeteria

**Comment by wendylamptey** [comment]
Let's make time to go to the cafeteria for more clarifications? What time do you all suggest?

**Comment by mitchiexx4** [comment]
What exactly did sir say we should work on? Can someone pls clarify for me

**Comment by wendylamptey** [comment]
He said our work is not well-fed so we should do it better

**Comment by wendylamptey** [comment]
We are to make changes to enable the cafeteria to update the menu and also we should use the form and make sure it works

**Comment by mitchiexx4** [comment]
Okay so what are we working on?

**Comment by mitchiexx4** [comment]
We should make sure the form works? And we should try to place orders?

**Comment by wendylamptey** [comment]
We need to find a way for the cafeteria to see orders so it could be on their phones

**Comment by wendylamptey** [comment]
And also our ghithub should be active because it's part of our marks

**Comment by wendylamptey** [comment]
And also our ghithub should be active because it's part of our marks

**Comment by mitchiexx4** [comment]
Okay that's not a problem. We know the form works already. Now we have to convince the cafeteria to use it. And they don't have a fixed menu so maybe make it possible for them to update the menu daily or?

**Comment by wendylamptey** [comment]
Yes so tomorrow if there's class let's all come so we go there after the class they will be less busy by then

**Comment by mitchiexx4** [comment]
Okay sure. Let's pray there's class

**Comment by SL12Golden** [comment]
Okay

**Comment by SL12Golden** [comment]
Do you all have a jotform account?

**Comment by mitchiexx4** [comment]
Yes I have one

**Comment by wendylamptey** [comment]
I have one

**Comment by SL12Golden** [comment]
Can you please send me your jotform account so that I can create a team on jotform

**Comment by mitchiexx4** [comment]
The name or the email used?

**Comment by SL12Golden** [comment]
Email

**Comment by mitchiexx4** [comment]
kplorlar17@gmail.com

**Comment by mitchiexx4** [comment]
Hi

**Comment by mitchiexx4** [comment]
Sheryl. Did you see our conversation on WhatsApp??

**Comment by mitchiexx4** [comment]
Please you need to be in school very early on Monday. Before class starts. I don't know what time the cafeteria will be open but we need to be in school very early. You especially

**Comment by mitchiexx4** [comment]
Since you're the creator of the form. I'm going to try and place an order now. Please let's all try it

**Comment by wendylamptey** [comment]
Alright

**Comment by mitchiexx4** [comment]
Sheryl, I placed an order. Can you see it ? Did you receive it?

**Comment by mitchiexx4** [comment]
When you're less busy, pls respond so I know okay?

**Comment by wendylamptey** [comment]
Sheryl please make the changes so the cafeteria can update the form anytime they want

**Comment by SL12Golden** [comment]
![20240405_173620.jpg](https://github.com/mitchiexx4/girls-group-for-IT/assets/147264526/3bf97d44-1769-4925-a8b7-10918f6eeab7)

I was able to view your orders

**Comment by mitchiexx4** [comment]
Okayy

**Comment by mitchiexx4** [comment]
Can you come early on Monday?

**Comment by wendylamptey** [comment]
Hello, since we couldn't even meet today I was suggesting we meet tomorrow early before lunch or after 4pm we can finalize the work.

**Comment by wendylamptey** [comment]
Sheryl how far with the report?

**Comment by SL12Golden** [comment]
![Cafeteria Report.docx](https://github.com/mitchiexx4/girls-group-for-IT/files/15053515/Cafeteria.Report.docx)

This is what I have done so far

**Comment by wendylamptey** [comment]
Oh okay, cool

**Comment by wendylamptey** [comment]
Are you using the right font size?

**Comment by wendylamptey** [comment]
Send the parts you haven't done yet

**Comment by SL12Golden** [comment]
2. Project Cost
4. Project Charter
5. Time Schedule
6. Implementation
8. Conclusion
I'm working on it

**Comment by SL12Golden** [comment]
> Are you using the right font size?

Yes
